wezterm-gui/src/termwindow/render.rs

```diff
@@ -817,6 +1104,15 @@ impl super::TermWindow {
             match item.item {
                 TabBarItem::LeftStatus => left_status.push(item_to_elem(item)),
                 TabBarItem::None | TabBarItem::RightStatus => right_eles.push(item_to_elem(item)),
+                TabBarItem::WindowHideButton
+                | TabBarItem::WindowMaximizeButton
+                | TabBarItem::WindowCloseButton => {
+                    if self.config.fancy_window_decorations.is_left && cfg!(not(windows)) {
```

Comment:
can we eliminate the `cfg!` portion of this from here and keep this purely in terms of the data we have available via `self.config` and the OS parameters?